src/common/users/users.controller.ts

```diff
@@ -1,14 +1,16 @@
-import { Controller, Get } from "@nestjs/common";
+import { Controller, Get, Version } from "@nestjs/common";
 
-@Controller("users")
+@Controller({ path: "users" })
 export class UsersController {
+  @Version("1")
   @Get()
   userInfo(): string {
```

Comment:
- the function name is not important here
- the controller path and request method is the main factor
- eg. in your case userInfo or userHistory function name is not important, both function serves value for {{version}}/users